app-config.yaml

```diff
@@ -373,6 +373,11 @@ auth:
       development: {}
 costInsights:
   engineerCost: 200000
+  baseCurrency:
```

Comment:
Sorry to be picky about this one but can we remove it entirely as this is optional. Just to keep our config slim 😅